pipeline/07-export.R

```diff
@@ -263,6 +263,15 @@ assessment_card <- dbGetQuery(
   ")
 )
 
+# Pull assessable permit flag
+flag_assessable_permits <- dbGetQuery(
+  conn = AWS_ATHENA_CONN_NOCTUA, glue("
+  SELECT pin, has_recent_assessable_permit
+  FROM default.vw_pin_status
```

Comment:
This won't actually work until https://github.com/ccao-data/data-architecture/pull/733 comes in, but I tested it against my development database and it works.